entrypoints/backend.sh

```diff
@@ -28,10 +28,10 @@ then
   /app/src/manage.py loaddata test_fixtures
 fi
 
-
+GUNICORN_COMMAND="gunicorn --chdir=src --reload --workers=12 --bind=0.0.0.0:8000 $@"
```

Comment:
do we want 12 workers for our dev environment?

Reply:
@jerbob broke this.

Reply:
No, we do not.